.vib/airflow/cypress/cypress/integration/airflow_test_suite.js

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+import { random } from "./utils";
+
+it("allows user to log in and log out", () => {
+  cy.login();
+  cy.contains("DAGs").should('be.visible');
+  cy.contains("div", "Invalid login").should("not.exist");
+
+  cy.visit("/logout").then(() => {
+    cy.clearCookies();
+    cy.reload();
+    cy.contains("DAGs").should('not.exist');
+  });
+
+});
+
+it("allows triggering execution of a sample DAG", () => {
+  cy.login();
+  cy.fixture("DAGs").then((dags) => {
+    const TRIGGERED_OK_MESSAGE = `Triggered ${dags.triggered.id}, it should start`;
+
+    cy.get("#dag_query")
+      .clear()
+      .type(`${dags.triggered.id}{enter}`);
+    cy.get(`a[href="/tree?dag_id=${dags.triggered.id}"]`).should("be.visible").click();
+    cy.get('a[aria-label="Trigger DAG"]').click();
+    cy.contains("Trigger DAG").click(); // A dropdown appears and clicking again is needed
```

Comment:
You can use an additional identifier, just to differentiate the two elements for the reader of the code. ex `cy.contains ('button,'Trigger DAG')`.

Reply:
Thanks for the suggestion! Included it!